lib/pr_summary.js

```diff
@@ -34,8 +34,16 @@ class PRSummary {
     cli.table('Title', `${title} (#${prid})`);
     const authorHint =
       this.data.authorIsNew() ? ', first-time contributor' : '';
-    cli.table('Author',
-      `${author.name} <${author.email}> (@${author.login}${authorHint})`);
+
+    if (author.name && author.email) {
+      cli.table('Author',
+        `${author.name} <${author.email}> (@${author.login}${authorHint})`);
+    } else {
+      // user do not have correct `git config` setup
```

Comment:
I think the information here comes from their github account settings? The `git config` setup are only visible in the commit information.

Take a look at the profile page https://github.com/antoine-amara and they don't have the name nor the email setup.

Reply:
Yeah actually not too sure about i guessed it since if you look at the issues the commit the user reference have correct config of commit.

Reply:
> Take a look at the profile page https://github.com/antoine-amara and they don't have the name nor the email setup.

I don't think that matters see the issue link and the commits that are referenced they seem to perfect commit except the last one (which was the one in the PR).

Reply:
I don't think it's possible to get an empty `git config`, since git should prompt for a username and a email when you commit for the first time. So the commits should always come with a name and a email (could be totally bogus, but still). It is possible to have empty github profile settings if you don't put those down during github account registration, or deletes them afterwards.

Reply:
See that something i am unsure about too. I have no idea how that occurred at the first place.

Reply:
It seems like the user squashed a that commit so it might be that he changed his email causing the commit to empty details. Since it does have users profile picture.

Reply:
@cPhost Hmm..maybe the user has set the email to private? I believe that would cause the email queried in the API to be `null` but Github still knows how to associate the commits with the account since they know about the private email address. Either way the warning should be something like "Could not retrieve the email from the PR author's GitHub profile"

Reply:
Ok, cool i can update the warning, also the name is also not present so i will add that in too.